_posts/2020-02-20-contractors.markdown

```diff
@@ -0,0 +1,213 @@
+---
+permalink: v1/contractors
+layout: sidebar
+title: Contractors
+---
+
+# Contractors
+
+## Attributes
+
+| Attribute                     | Type              | Read-Only | Optional | Default | Description
+| :----------                   |:-------------     |:---------:|:--------:|:--------|:-------------
+| `id`                          | Integer           |     X     |          |         | The unique identifier of this contractor in Gusto.
+| `version`                     | String            |     X     |          |         | The version of this object. See <a href="/v1/considerations/versioning">the versioning documentation</a> for a more in depth explanation of versions.
+| `company_id`                  | Integer           |     X     |          |         | The ID of the company to which the contractor belongs.
+| `type`                        | String            |           |          |         | The contractor type, either an "Individual" or a "Business".
+| `start_date`                  | String            |           |          |         | The day when the contractor will start working for the company.
+| `is_active`                   | Boolean           |     X     |          |         | The contractor's status with the company. Contractors created through the API will default to active.
+| `first_name`                  | String            |           |          |         | The contractor's first name. This attribute is required for "Individual" contractors and will be ignored for "Business" contractors.
+| `middle_initial`              | String            |           |    X     | null    | The contractor's middle initial. This attribute is optional for "Individual" contractors and will be ignored for "Business" contractors.
+| `last_name`                   | String            |           |          |         | The contractor's last name. This attribute is required for "Individual" contractors and will be ignored for "Business" contractors.
+| `email`                       | String            |           |    X     | null    | The contractor's email address. This attribute is optional for "Individual" contractors and will be ignored for "Business" contractors. This attribute should only be included in POST requests.
```

Comment:
I would expand on this a bit to call out what happens outside of a POST request. For example:

> If you include `email` in a PUT request, it will be ignored.